components/typography.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Text } from 'react-native-paper'
+import { StyleSheet } from 'react-native'
+
+export function Headline({ children, color, style, ...props }) {
+  return (
+    <Text
+      style={[styles.headline, color && { color }, color && { color }, style]}
+      {...props}
+    >
+      {children}
+    </Text>
+  )
+}
+
+export function BodyText({ children, color, style, ...props }) {
+  return (
+    <Text
+      style={[styles.bodyText, color && { color }, color && { color }, style]}
+      {...props}
+    >
+      {children}
+    </Text>
+  )
+}
+
+const styles = StyleSheet.create({
+  headline: {
+    fontFamily: 'Poppins_700Bold',
+    fontWeight: '700',
+    fontSize: 60,
+  },
+  bodyText: {
+    fontFamily: 'DidactGothic_400Regular',
+    fontWeight: '400',
+    fontSize: 16,
+    lineHeight: 24,
+  },
+})
```

Comment:
important thing to note. if we need custom fonts, we can't use RN's Text anymore. We must use either a custom set of typograpy components (as you're doing here), or change the font configuration of paper's typography components to use our fonts. see https://callstack.github.io/react-native-paper/theming.html and the source code of paper's default theme to see how font customization is done https://github.com/callstack/react-native-paper/blob/main/src/styles/DefaultTheme.tsx

Reply:
Do you think it's worth me sticking that in a comment in this file?

Reply:
sure. my comment is not exactly accurate though, it's not true that we can't use Text anymore, but each time we use it we would need to style it. I would prefer to have a set of typography components instead, or use those in paper, after customizing paper's fonts in the theme

Reply:
Sounds like something we can disclose at the project README file. Thoughts?

Reply:
I think we could definitely do with some documentation (in general).